server/sonar-server/src/main/java/org/sonar/server/qualityprofile/QProfileRef.java

```diff
@@ -0,0 +1,155 @@
+/*
+ * SonarQube
+ * Copyright (C) 2009-2016 SonarSource SA
+ * mailto:contact AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+package org.sonar.server.qualityprofile;
+
+import javax.annotation.Nullable;
+import org.sonar.api.resources.Languages;
+import org.sonar.api.server.ws.Request;
+import org.sonar.api.server.ws.WebService;
+import org.sonar.server.component.ws.LanguageParamUtils;
+
+import static com.google.common.base.Preconditions.checkArgument;
+import static com.google.common.base.Preconditions.checkState;
+import static java.util.Objects.requireNonNull;
+import static org.apache.commons.lang.StringUtils.isEmpty;
+
+/**
+ * Reference to a Quality profile. The two exclusive options to reference a profile are:
+ * <ul>
+ *   <li>by its key</li>
+ *   <li>by the couple {language, name}</li>
+ * </ul>
+ */
+public class QProfileRef {
+
+  public static final String PARAM_LANGUAGE = "language";
```

Comment:
yes but that was not the case before this PR :-) Constants were in QProfileIdentificationParamUtils. 
Still I'm not sure about the value of your proposal. I'd like definition and loading of these parameters to be in the same class as they are tightly coupled.